methods.ts

```diff
@@ -67,7 +70,7 @@ export type ApiMethods<F> = {
     limit?: number;
     /** Timeout in seconds for long polling. Defaults to 0, i.e. usual short polling. Should be positive, short polling should be used for testing purposes only. */
     timeout?: number;
-    /** A list of the update types you want your bot to receive. For example, specify [“message”, “edited_channel_post”, “callback_query”] to only receive updates of these types. See Update for a complete list of available update types. Specify an empty list to receive all update types except chat_member (default). If not specified, the previous setting will be used.
+    /** A list of the update types you want your bot to receive. For example, specify ["message", "edited_channel_post", "callback_query"] to only receive updates of these types. See Update for a complete list of available update types. Specify an empty list to receive all update types except chat_member, message_reaction, and message_reaction_count (default). If not specified, the previous setting will be used.
 
     Please note that this parameter doesn't affect updates created before the call to the getUpdates, so unwanted updates may be received for a short period of time. */
     allowed_updates?: ReadonlyArray<Exclude<keyof Update, "update_id">>;
```

Comment:
question (non-blocking): can't we use the available updates type here?

Reply:
Those from the core lib?

Reply:
oh, I got confused, I thought I was looking at the core lib 🤪